frontend/lobby/waiting/waiting.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+let params = new URLSearchParams(document.location.search.substring(1));
+const gameID = params.get("gameID");
+const playerID = params.get("playerID");
+const gameURL = `/game/?gameID=${gameID}&playerID=${playerID}`;
+
+function checkForOpponent() {
+    var req = new XMLHttpRequest();
+    req.open("GET", `http://backend:3000/games/${gameID}/player/${playerID}`);
+    req.responseType = "json";
+    
+    req.onload = function () {
+        const status = req.response["status"];
+        
+        if (status !== "Waiting for an opponent") {
```

Comment:
Couldn't we solve #43 on this merge?

Reply:
That is a whole different lot of work that will imply both backend and frontend changes and probably some sort of `config.yml` file at the root.